internal/spoa.go

```diff
@@ -291,7 +234,7 @@ func (s *SPOA) processRequest(spoeMsg *spoe.Message) ([]spoe.Action, error) {
 
 	tx = app.waf.NewTransactionWithID(req.id)
 	if tx.IsRuleEngineOff() {
-		app.logger.Warn("Rule engine is Off, Coraza is not going to process any rule")
+		log.Warn().Msg("Rule engine is Off, Coraza is not going to process any rule")
 		return s.message(miss), nil
 	}
 
```

Comment:
Log in application log too before throwing to criteo

Reply:
Find where `txn.coraza.error` is set.

Make constant error message like allowMessage. In case of error return that message here instead of criteo (if criteo is responsible for error messages now). Let criteo send error messages in case of his fail.

Reply:
https://github.com/corazawaf/coraza-spoa/blob/dd5eb86d1e9abbdd5fe568249f36a6d85257eba7/docker/haproxy/coraza.cfg#L5-L6

https://github.com/haproxy/haproxy/blob/49ddd87d41b62797d50d7c59e843d0c0b0236eb5/doc/SPOE.txt#L300